### PR TITLE
Add release notes for Frontend v3.13.1 release

### PR DESCRIPTION
**Pull request by EoinShaughnessy**
Closes [#2315](https://github.com/alphagov/govuk-frontend/issues/2315).

### CODE REVIEW

**Review by vanitabarrett — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,17 +2,19 @@
...
+## 3.13.1 (Patch release)
+
 ### Fixes
 
 We’ve made fixes to GOV.UK Frontend in the following pull requests:
 
-- [#2264: Improve focus state for radio and checkbox controls in forced color mode (for example Windows High Contrast Mode)](https://github.com/alphagov/govuk-frontend/pull/2264) – thanks to [@adamliptrot-oc](https://github.com/adamliptrot-oc) for reporting this issue.
-- [#2265: Don’t remove focus outline from disabled link buttons in forced color mode](https://github.com/alphagov/govuk-frontend/pull/2265)
-- [#2273: Fix invisible footer OGL logo in forced color mode](https://github.com/alphagov/govuk-frontend/pull/2273) – thanks to [@oscarduignan](https://github.com/oscarduignan) for reporting this issue.
+- [#2264: Improve focus state for radio and checkbox controls in forced color mode (for example, Windows High Contrast Mode)](https://github.com/alphagov/govuk-frontend/pull/2264) – thanks to [@adamliptrot-oc](https://github.com/adamliptrot-oc) for reporting this issue.
```
Comment (PippaClarkGDS) [inline CHANGELOG.md:11]:
```suggestion
- [#2264: Improve focus state for radio and checkbox controls in forced color mode (for example, Windows High Contrast Mode)](https://github.com/alphagov/govuk-frontend/pull/2264) – thanks to [@adamliptrot-oc](https://github.com/adamliptrot-oc) for reporting this issue
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,17 +2,19 @@
...
+- [#2265: Do not remove focus outline from disabled link buttons in forced color mode](https://github.com/alphagov/govuk-frontend/pull/2265)
+- [#2273: Fix invisible footer on Open Government Licence logo in forced color mode](https://github.com/alphagov/govuk-frontend/pull/2273) – thanks to [@oscarduignan](https://github.com/oscarduignan) for reporting this issue.
```
Comment (PippaClarkGDS) [inline CHANGELOG.md:13]:
```suggestion
- [#2273: Fix invisible footer on Open Government Licence logo in forced color mode](https://github.com/alphagov/govuk-frontend/pull/2273) – thanks to [@oscarduignan](https://github.com/oscarduignan) for reporting this issue
```

**Review by PippaClarkGDS — APPROVED** [review]
Looks good - 2 small style changes

**Comment by EoinShaughnessy** [comment]
Thanks, @PippaClarkGDS!

**Comment by EoinShaughnessy** [comment]
Have sent notes for 2i.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,17 +2,19 @@
...
+## 3.13.1 (Patch release)
+
 ### Fixes
 
 We’ve made fixes to GOV.UK Frontend in the following pull requests:
 
-- [#2264: Improve focus state for radio and checkbox controls in forced color mode (for example Windows High Contrast Mode)](https://github.com/alphagov/govuk-frontend/pull/2264) – thanks to [@adamliptrot-oc](https://github.com/adamliptrot-oc) for reporting this issue.
-- [#2265: Don’t remove focus outline from disabled link buttons in forced color mode](https://github.com/alphagov/govuk-frontend/pull/2265)
-- [#2273: Fix invisible footer OGL logo in forced color mode](https://github.com/alphagov/govuk-frontend/pull/2273) – thanks to [@oscarduignan](https://github.com/oscarduignan) for reporting this issue.
+- [#2264: Improve focus state for radio and checkbox controls in forced color mode (for example, Windows High Contrast Mode)](https://github.com/alphagov/govuk-frontend/pull/2264) – thanks to [@adamliptrot-oc](https://github.com/adamliptrot-oc) for reporting this issue
```
Comment (cmcnallygds) [inline CHANGELOG.md:11]: I think it should either be 'forced-color mode' or 'forced colors mode' but definitely double check this :)

**Review by cmcnallygds — COMMENTED** [review]
Have made a small suggestion!

File: CHANGELOG.md (hunk 3 above)
Comment (EoinShaughnessy) [reply CHANGELOG.md:11]: Great spot! Just looked up [Chrome](https://www.chromestatus.com/feature/5757293075365888#:~:text=Adds%20the%20'forced%2Dcolors',color%20palette%20on%20the%20page.) and [Mozilla](https://developer.mozilla.org/en-US/docs/Web/CSS/forced-color-adjust), and yep, 'forced colors mode' it is..I'll make the needed changes here and then reassign you the review if that's ok. Thanks again!

Comment (EoinShaughnessy) [inline CHANGELOG.md:11]:
```suggestion
- [#2264: Improve focus state for radio and checkbox controls in forced colors mode (for example, Windows High Contrast Mode)](https://github.com/alphagov/govuk-frontend/pull/2264) – thanks to [@adamliptrot-oc](https://github.com/adamliptrot-oc) for reporting this issue
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,17 +2,19 @@
...
+- [#2265: Do not remove focus outline from disabled link buttons in forced color mode](https://github.com/alphagov/govuk-frontend/pull/2265)
```
Comment (EoinShaughnessy) [inline CHANGELOG.md:12]:
```suggestion
- [#2265: Do not remove focus outline from disabled link buttons in forced colors mode](https://github.com/alphagov/govuk-frontend/pull/2265)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,17 +2,19 @@
...
+- [#2273: Fix invisible footer on Open Government Licence logo in forced color mode](https://github.com/alphagov/govuk-frontend/pull/2273) – thanks to [@oscarduignan](https://github.com/oscarduignan) for reporting this issue
```
Comment (EoinShaughnessy) [inline CHANGELOG.md:13]:
```suggestion
- [#2273: Fix invisible footer on Open Government Licence logo in forced colors mode](https://github.com/alphagov/govuk-frontend/pull/2273) – thanks to [@oscarduignan](https://github.com/oscarduignan) for reporting this issue
```

File: CHANGELOG.md (hunk 3 above)
Comment (EoinShaughnessy) [reply CHANGELOG.md:11]:
```suggestion
- [#2264: Improve focus state for radio and checkbox controls in forced colors mode (for example, Windows High Contrast Mode)](https://github.com/alphagov/govuk-frontend/pull/2264) – thanks to [@adamliptrot-oc](https://github.com/adamliptrot-oc) for reporting this issue
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,17 +2,19 @@
...
 - [#2277: Fix invisible start button chevron in forced color mode](https://github.com/alphagov/govuk-frontend/pull/2277)
```
Comment (EoinShaughnessy) [inline CHANGELOG.md:14]:
```suggestion
- [#2277: Fix invisible start button chevron in forced colors mode](https://github.com/alphagov/govuk-frontend/pull/2277)
```

**Review by cmcnallygds — APPROVED** [review]
Looks good

**Comment by EoinShaughnessy** [comment]
Content has passed 2i. I'll squash its commits and update the Kanban board.

File: CHANGELOG.md [hunk 7]
```diff
@@ -2,17 +2,19 @@
 
 ## Unreleased
 
+## 3.13.1 (Patch release)
```
Comment (EoinShaughnessy) [inline CHANGELOG.md:5]:
```suggestion
## 3.13.1 (Fix release)
```

**Pull request merged**